src/main/kotlin/org/rust/ide/annotator/fixes/NameSuggestionFix.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement
+import com.intellij.grazie.speller.utils.Distances.levenshtein
```

Comment:
Oh, I think we shouldn't use this implementation because it's from `grazie-spellchecker` lib and I doubt that we can rely on it because it can be changed, doesn't exist in all IDEs or even excluded from IDEA/CLion libraries in the future. Let's better use `com.intellij.util.text.EditDistance#levenshtein` that a part of common API

Reply:
Oops, wrong autoimport, thanks.